templates/pages/product.html

```diff
@@ -19,6 +19,8 @@
     <div itemscope itemtype="http://schema.org/Product">
         {{> components/products/product-view schema=true  }}
 
+        {{{region name="page_content"}}}
```

Comment:
Could this be better named? I think consumers would want to know the relationship between this region and other important PDP components such as:

1. Product Price
2. Product Reviews
3. Product Description
4. Add-to-cart button

Reply:
I might tend to disagree with building regions with very specific naming unless we have a very specific use case to it. I think we should keep them generic but they should be better named. The reason for keeping it generic is to make them accessible across multiple templates.